src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        for (int i = 0; i < 3; i++) {
+            figure[i] = figureSupplier.getRandomFigure();
+            figure[i].setColor(colorSupplier.getRandomColor());
+        }
+        for (int i = 3; i < 6; i++) {
+            figure[i] = figureSupplier.getDefaultFigure();
+        }
+        for (int i = 0; i < 6; i++) {
+            figure[i].drawFigure();
```

Comment:
The method 'drawFigure' does not follow the Java naming conventions for methods. It should be 'draw' instead, as method names are typically verbs or verb phrases.